www/src/templates/template-api-markdown.js

```diff
@@ -0,0 +1,258 @@
+              {page.frontmatter.title}
+            </h1>
+          </Container>
+          <Container
+            overrideCSS={{
+              pt: 0,
+              position: `static`,
+              [mediaQueries.lg]: {
+                pb: 9,
+              },
+              [toc && mediaQueries.xl]: {
+                ...containerStyles,
+                display: `flex`,
+                alignItems: `flex-start`,
+              },
+            }}
+          >
+            {toc && (
+              <div
+                sx={{
+                  order: 2,
+                  [mediaQueries.xl]: {
+                    ml: 9,
+                    maxWidth: `tocWidth`,
+                    position: `sticky`,
+                    top: t =>
+                      `calc(${t.sizes.headerHeight} + ${
+                        t.sizes.bannerHeight
+                      } + ${t.space[9]})`,
+                    maxHeight: t =>
+                      `calc(100vh - ${t.sizes.headerHeight} - ${
+                        t.sizes.bannerHeight
+                      } - ${t.space[9]} - ${t.space[9]})`,
+                    overflow: `auto`,
+                  },
+                }}
+              >
+                <TableOfContents location={location} page={page} />
+              </div>
+            )}
+            <div
+              sx={{
+                [page.tableOfContents.items && mediaQueries.xl]: {
+                  maxWidth: `mainContentWidth.withSidebar`,
+                  minWidth: 0,
+                },
+              }}
+            >
+              <div>
+                <MDXRenderer
+                  slug={page.fields.slug}
+                  mergedFuncs={mergeFunctions(data, pageContext)}
+                >
+                  {page.body}
+                </MDXRenderer>
+                {page.frontmatter.issue && (
+                  <a
+                    href={page.frontmatter.issue}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    See the issue relating to this stub on GitHub
+                  </a>
+                )}
+                <PrevAndNext sx={{ mt: 9 }} prev={prev} next={next} />
+                <MarkdownPageFooter page={page} />
+              </div>
+            </div>
+          </Container>
+        </DocSearchContent>
+        <FooterLinks />
+      </Layout>
+    </React.Fragment>
+  )
+}
+
+export default DocsTemplate
+
+export const pageQuery = graphql`
+  query($path: String!, $jsdoc: String, $apicalls: String) {
+    mdx(fields: { slug: { eq: $path } }) {
+      body
+      excerpt
+      timeToRead
+      tableOfContents
+      fields {
+        slug
+        anchor
+      }
+      frontmatter {
+        title
+        overview
+        issue
+        disableTableOfContents
+        tableOfContentsDepth
+      }
+      ...MarkdownPageFooterMdx
+    }
+    jsdoc: allFile(filter: { relativePath: { regex: $jsdoc } }) {
```

Comment:
If we don't want to do regex maybe we can do

```
jsdoc:
  eq: "path/to/file"
```

or

```
jsdoc:
  is: [/path/one, /path/two]
```

Or in `gatsby-node` do:

```
const { jsdoc } = node.frontmatter.jsdoc
{
  jsdoc: typeof jsdoc === 'string' ? { eq: jsdoc } : { in: jsdoc }
}
```

Reply:
I am planning to always query using `in:`, but we can provide a single string fallback by inserting it into a single element list in `gatsby-node` when creating context.